### PR TITLE
Add Tests For Configurable Frequency

### PR DESCRIPTION
**Pull request by EmilyYLMa**
- Added tests for impedance calculation at 50Hz
- Tests are based on results from Cyme

### CODE REVIEW

File: tests/test_carsons.py [hunk 1]
```diff
@@ -11,11 +11,16 @@
...
 # network.
-OHM_PER_MILE_TO_OHM_PER_METER = 1 / 1609.344
+OHM_PER_MILE_TO_OHM_PER_METER = 1 / 1_609.344
+OHM_PER_KILOMETER_TO_OHM_PER_METER = 1 / 1_000
 
 
 class ACBN_geometry_line():
     """ IEEE 13 Configuration 601 Line Geometry """
+
+    def __init__(self, **kwargs):
```
Comment (AnjoMan) [inline tests/test_carsons.py]: should just do `Foo(self, ƒ=60)`

File: tests/test_carsons.py [hunk 2]
```diff
@@ -56,6 +61,9 @@ def phases(self):
 class CBN_geometry_line():
     """ IEEE 13 Configuration 603 Line Geometry """
 
+    def __init__(self, **kwargs):
```
Comment (AnjoMan) [inline tests/test_carsons.py]: here too

**Comment by AnjoMan** [comment]
addresses #26

File: tests/test_carsons.py (hunk 1 above)
Comment (EmilyYLMa) [reply tests/test_carsons.py]: Done

File: tests/test_carsons.py (hunk 2 above)
Comment (EmilyYLMa) [reply tests/test_carsons.py]: Done

**Pull request merged**